synapse/types/__init__.py

```diff
@@ -95,6 +95,7 @@
 #
 # Unlike StrCollection, StrSequence is an ordered collection of strings.
 StrSequence = Union[Tuple[str, ...], List[str]]
+BytesSequence = Union[Tuple[bytes, ...], List[bytes]]
```

Comment:
I guess this is because a `bytes` instance is considered a `Sequence[bytes]`?

Reply:
Actually...it isn't, it is a `Sequence[int]` so maybe we don't need this.

Reply:
Aha!

```python
>>> list(b"123")
[49, 50, 51]
```

Reply:
Thanks, I forgot this isn't quite the same, even though I wrote a blog post about it. 🤦